virtwho/config.py

```diff
@@ -18,21 +18,30 @@
 Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 """
 
+import collections
 import os
-
 from ConfigParser import SafeConfigParser, NoOptionError, Error, MissingSectionHeaderError
-from virtwho import DefaultInterval
+from virtwho import log
 from password import Password
 from binascii import unhexlify
 import hashlib
 import json
+import six
 import util
 
-VIRTWHO_CONF_DIR = "/etc/virt-who.d/"
+# Module-level logger
+logger = log.getLogger('config', queue=False)
+
+VW_CONF_DIR = "/etc/virt-who.d/"
 VIRTWHO_TYPES = ("libvirt", "vdsm", "esx", "rhevm", "hyperv", "fake", "xen")
 VIRTWHO_GENERAL_CONF_PATH = "/etc/virt-who.conf"
-VIRTWHO_GLOBAL_SECTION_NAME = "global"
+VW_GLOBAL = "global"
```

Comment:
`VW` stands for Volkswagen, right? ;-) ... Seriously, I would use VW in other constants too.